priv/templates/phx.gen.auth/auth.ex

```diff
@@ -70,7 +71,15 @@ defmodule <%= inspect auth_module %> do
   #       |> put_session(:preferred_locale, preferred_locale)
   #     end
   #
-  defp renew_session(conn) do
+  defp renew_session(conn, <%= schema.singular %> \\ nil)
```

Comment:
I personally don't like using default values for private functions. Unless in this case there is no guarantee we always have a user?

Reply:
changed it to pass nil in logout and removed the default value